sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/sonarqube.yaml

```diff
@@ -2,30 +2,6 @@ provider:
   metadata:
     name: SonarQube
     category: Code analysis
-    references:
-      - description: SonarQube - Generating and using tokens
-        link: https://docs.sonarqube.org/latest/user-guide/user-account/generating-and-using-tokens/
-        type: Documentation
-      - description: SonarCloud - User Accounts - User tokens
-        link: https://docs.sonarcloud.io/advanced-setup/user-accounts/#user-tokens
-        type: Documentation
-      - description: OWASP Top 10 2021 Category A7 - Identification and Authentication Failures
-        link: https://owasp.org/Top10/A07_2021-Identification_and_Authentication_Failures/
-        type: Standards
-      - description: OWASP Top 10 2017 Category A3 - Sensitive Data Exposure
-        link: https://www.owasp.org/www-project-top-ten/2017/A3_2017-Sensitive_Data_Exposure
-        type: Standards
-      - description: MITRE, CWE-798 - Use of Hard-coded Credentials
-        link: https://cwe.mitre.org/data/definitions/798
-        type: Standards
-      - description: MITRE, CWE-259 - Use of Hard-coded Password
-        link: https://cwe.mitre.org/data/definitions/259
-        type: Standards
-      - description: SANS Top 25 - Porous Defenses
-        link: https://www.sans.org/top25-software-errors/#cat3
-        type: Standards
-    message: Make sure this SonarQube token gets revoked, changed, and removed from the code.
```

Comment:
Missing the message, didn't see that. You won't be able to push while it's deleted